base_site/index.html

```diff
@@ -1,96 +1,222 @@
+                    </a>
+                  </div>
+                  <div id="w-node-e83de36d-198e-e006-eaa4-76da440cbd76-c2a09c1d" class="product-list_image-wrapper"><img src="images/hsized.png" loading="lazy" alt="" class="product-list_image"></div>
+                </div>
+                <div class="w-layout-grid product-list_component">
+                  <div class="product-list_content">
+                    <div class="product_logo-wrap is-tket"><img src="images/tket.svg" loading="lazy" alt="" class="image-fit-contain"></div>
+                    <div class="spacer-small"></div>
+                    <p class="product_description">Hardware agnostic quantum compiler</p>
+                    <div class="spacer-medium"></div>
+                    <a href="https://docs.quantinuum.com/tket" class="product_link-wrap w-inline-block">
+                      <p class="product_link-text">Get Started with TKET</p>
+                      <p class="product_description is-list">Get started now with our open-source  Python library, pyTKET</p>
+                    </a>
+                    <div class="spacer-small"></div>
+                    <a href="https://docs.quantinuum.com/tket/user-manual/" class="product_link-wrap w-inline-block">
+                      <p class="product_link-text">Tutorials and Documentation</p>
+                      <p class="product_description is-list">Find the latest technical documentation and additional resources</p>
+                    </a>
+                  </div>
+                  <div id="w-node-f0c4217f-9d1f-0ed9-1d1c-8741d622147b-c2a09c1d" class="product-list_image-wrapper"><img src="images/tket-image.png" loading="lazy" alt="" class="product-list_image"></div>
+                </div>
+                <div class="w-layout-grid product-list_component">
+                  <div class="product-list_content">
+                    <div class="product_logo-wrap"><img src="images/inquanto.svg" loading="lazy" alt="" class="image-fit-contain"></div>
+                    <div class="spacer-small"></div>
+                    <p class="product_description">State-of-the-art quantum chemistry platform for complex molecular and materials simulations</p>
+                    <div class="spacer-medium"></div>
+                    <a href="https://docs.quantinuum.com/inquanto/index.html" class="product_link-wrap w-inline-block">
+                      <p class="product_link-text">Get Started with InQuanto</p>
+                      <p class="product_description is-list">Learn how to accelerate quantum  computational chemistry</p>
+                    </a>
+                    <div class="spacer-small"></div>
+                    <a href="https://docs.quantinuum.com/inquanto/manual/howto.html" class="product_link-wrap w-inline-block">
+                      <p class="product_link-text">Tutorials and Documentation</p>
```

Comment:
```suggestion
                    <a href="https://docs.quantinuum.com/inquanto/manual/howto.html" class="product_link-wrap w-inline-block">
                      <p class="product_link-text">Get Started with InQuanto</p>
                      <p class="product_description is-list">Learn how to accelerate quantum  computational chemistry</p>
                    </a>
                    <div class="spacer-small"></div>
                    <a href="https://docs.quantinuum.com/inquanto/index.html" class="product_link-wrap w-inline-block">
                      <p class="product_link-text">Tutorials and Documentation</p>
```

Swap the "Get started" and "Tutorials and Documentation" links.